Plugins/Public/base_plugin/CoreModule.cpp

```diff
@@ -226,12 +226,20 @@ void CoreModule::SetShieldState(const int shieldState)
 bool CoreModule::Timer(uint time)
 {
 	// Disable shield if time elapsed
-	if (base->shield_timeout < time)
+	if (base->shield_timeout && base->shield_timeout < time)
 	{
 		base->shield_timeout = 0;
 		base->shield_state = PlayerBase::SHIELD_STATE_ONLINE;
 		SetShieldState(base->shield_state);
 	}
+	
+	// we need to periodically set the health of all POBs to trigger a clientside 'refresh'
```

Comment:
I'm sure we did this in another PR - have you rebased?

Reply:
resolved